### PR TITLE
Simple Features to Add

### PR DESCRIPTION
**Pull request by philovivero**
Minor Features PR

- Verbiage around closing, the nag and post-close messaging. Trying to make it way more obvious to engineers to close the incident when mitigated/repaired, even if there's more chat in the channel afterward. Added new verbs as well.

### CODE REVIEW

File: slack/incident_commands.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def set_severity(incident: Incident, user_id: str, message: str):
     return True, None
 
 
-@incident_command(['reopen'], helptext="Re-open incident if closed.")
+@incident_command(['reopen','unrepaired'], helptext="Re-open incident if closed.")
```
Comment (fraenkel) [inline slack/incident_commands.py]: why would unrepaired re-open an incident?

File: slack/incident_commands.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def reopen_incident(incident: Incident, user_id: str, message: str):
     return True, None
 
 
-@incident_command(['close'], helptext='`[dup RCCA-ID]` Close this incident optionally as a duplicate of another incident.')
+@incident_command(['close','repair','repaired'], helptext='`[dup RCCA-ID]` Close this incident optionally as a duplicate of another incident.')
```
Comment (fraenkel) [inline slack/incident_commands.py]: why would repair close an incident? repaired is documented below so it makes sense.

File: slack/incident_notifications.py [hunk 3]
```diff
@@ -32,6 +32,7 @@ def statuspage_reminder(incident: Incident):
         comms_channel = CommsChannel.objects.get(incident=incident)
         comms_verbiage = "Does anyone know the incident's customer impact?\n"
         comms_verbiage += "> :white_small_square: Update StatusPage if there is customer impact: https://manage.statuspage.io/pages/3yz1tdpdnjl3/incidents\n"
+        comms_verbiage += "> :white_small_square: Page COPs liberally if you think particular customers may need to know.\n"
```
Comment (fraenkel) [inline slack/incident_notifications.py]: Provide the command to type?

File: slack/incident_commands.py (hunk 2 above)
Comment (philovivero) [reply slack/incident_commands.py]: I think I agree, removing `repair` verb.

File: slack/incident_notifications.py (hunk 3 above)
Comment (philovivero) [reply slack/incident_notifications.py]: Sure, why not. Adding this as another commit shortly.

File: slack/incident_commands.py (hunk 1 above)
Comment (philovivero) [reply slack/incident_commands.py]: It's just the opposite of `repaired` - but admittedly these verbs are a bit strained. The only reason we use `repair` and its variations is there's a MTTR (mean time to repair) we're tracking for incidents, so Range wants to use that word specifically to help when we start using that acronym.

**Pull request merged**